virtcontainers/qemu_arch_base.go

```diff
@@ -110,6 +111,12 @@ const (
 	maxDevIDSize            = 31
 )
 
+// This is the PCI start address assigned to the first bridge that
+// is added on the qemu command line. In case of x86_64, the first two PCI
+// addresses (0 and 1) are used by the platform while in case of ARM, address
+// 0 is reserved.
```

Comment:
Good comment, but if the behaviour is different for x86_64 vs ARM, why is the (x86_64-specific) constant in the generic base file?

Reply:
We just need to make sure that the address does not conflict. So I chose greater of the two addresses.